pkg/antctl/raw/test/command.go

```diff
@@ -0,0 +1,522 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/spf13/cobra"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var Command *cobra.Command
+var (
+	contextName string
+	k8sClient   *Client
+)
+
+func init() {
+	client, err := NewClient(contextName, "")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to create kubernetes client: %s", err)
+	}
+	k8sClient = client
+
+	parameters := Parameters{
+		antreaNamespace: "kube-system",
+		Writer:          os.Stdout,
+	}
+	Command = &cobra.Command{
+		Use:   "test",
+		Short: "test connectivity",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Print("test starting")
+			check := NewAntreaConnectivityCheck(k8sClient, parameters)
+			return check.Run(context.Background())
+		},
+	}
+	Command.Flags().StringVarP(&parameters.antreaNamespace, "namespace", "n", "kube-system", "Configure namespace in which Antrea is running")
+	Command.Flags().BoolVar(&parameters.ForceDeploy, "force-deploy", false, "Force deploys test deployments")
+	//TODO
+	Command.Flags().BoolVar(&parameters.SingleNode, "single-node", false, "Runs tests applicable only on single node environment")
```

Comment:
If this is not used yet, we should just remove it for now

Reply:
alright sir